filter-ops.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
New file, let's you filter down a given schema to only what you want it to be

Reply:
I love the amount of yeeted code :)